app/views/notes/_comments.html.erb

```diff
@@ -15,10 +15,10 @@
     </div>
 
     <% if current_user %>
-      <%= render :partial => "comments/form", :locals => { title: I18n.t('notes._comments.post_comment'), comment: false, placeholder: I18n.t('notes._comments.post_placeholder') } %>
+      <%= render :partial => "comments/form", :locals => { title: I18n.translation('notes._comments.post_comment'), comment: false, placeholder: I18n.translation('notes._comments.post_placeholder') } %>
```

Comment:
The other changes are fine, but it looks like `I18n.translation`. Can you change these two instances back to `I18n.t`?

Reply:
Ok I am changing it.